### ISSUE TITLE
eval mode failure

### ISSUE THREAD

**Issue by TwiceMao** [NVlabs/InstantSplat#37]
Thanks for your awesome work. I can't successfully run the [preprocessed ](https://drive.google.com/file/d/1Z17tIgufz7-eZ-W0md_jUlxq89CD1e5s/view)TT dataset you provided  in eval mode. Could you please provide some help?

**Comment by kairunwen** [comment]
Hi. What was the error message there? It will be helpful in locating the errors given a detailed description.

**Comment by TwiceMao** [comment]
Hi, my custom test set is separate from the training set. The test set has never appeared in the training set. I want to synthesize a novel view on the test set (need to search for the corresponding camera pose based on the test color images), but according to the TT dataset organization you provided, running your eval mode code, the code does not execute the logic of new view synthesis. Cloud you please help me

**Comment by kairunwen** [comment]
We have updated the code with the preprint accordingly. 

- The camera parameters of training views are saved in `<source_path>/sparse_${n_views}/0`.
- The camera parameters of testing views are saved in `<source_path>/sparse_${n_views}/1`.

Please try re-running `bash scripts/run_eval.sh` and let me know if you need any further assistance.
